proto/sentry_protos/snuba/v1/request_common.proto

```diff
@@ -53,14 +54,6 @@ enum TraceItemName {
   TRACE_ITEM_NAME_EAP_ERRORS = 2;
 }
 
-enum TraceItemType {
-  TRACE_ITEM_TYPE_UNSPECIFIED = 0;
-  TRACE_ITEM_TYPE_SPAN = 1;
-  TRACE_ITEM_TYPE_ERROR = 2;
-  TRACE_ITEM_TYPE_LOG = 3;
-  TRACE_ITEM_TYPE_UPTIME_CHECK = 4;
-}
-
```

Comment:
I think this breaks backwards compatibility

Reply:
It does, I'll revert.